sdk/communication/AzureCommunicationCommon/AzureCommunication.podspec.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "AzureCommunication",
```

Comment:
sdk/communication/AzureCommunicationCommon/AzureCommunication.podspec.json

Should the podspec be renamed to `AzureCommunicationCommon.podspec.json`

Reply:
Oh sorry my mistake this is podspec marking it as deprecated